src/main/java/com/redhat/devtools/intellij/tektoncd/listener/TreeExpansionListener.java

```diff
@@ -23,6 +26,7 @@ public void treeWillExpand(TreeExpansionEvent treeExpansionEvent) {
         ParentableNode<? extends ParentableNode<?>> expandingElement = StructureTreeAction.getElement(treeExpansionEvent.getPath().getLastPathComponent());
         if (WatchHandler.get().canBeWatched(expandingElement)) {
             WatchHandler.get().setWatchByNode(expandingElement, treeExpansionEvent.getPath());
+            ((TektonTreeStructure)((Tree)treeExpansionEvent.getSource()).getClientProperty(Constants.STRUCTURE_PROPERTY)).fireModified(expandingElement);
```

Comment:
I don't like this patch at all because it seems to me that refresh will be cause each time an expand is requested, defeating the notion of cache. And it links two events that should not (expand and state (refresh).